magellan-library/build.gradle

```diff
@@ -42,8 +42,7 @@ android {
 }
 
 dependencies {
-
-  implementation project(':magellan-legacy')
```

Comment:
no more dependency on legacy ✨

Reply:
Why not move these into Magellan library and have legacy depend on that? That way we can avoid creating another module, and we're going to need that dependency when we make the compatibility navigator in legacy

Reply:
I was thinking we would make the compatibility navigator in magellan-support. But i suppose this works too.

Reply:
Done